### ISSUE TITLE
Bug:"/" inside repository name will affect image tag for scan

### ISSUE THREAD

**Issue by cageyv** [fivexl/aws-ecr-client-golang#10]
We can't use "/" inside docker tags
```TagRegexp = match(`[\w][\w.-]{0,127}`)```

**Comment by cageyv** [comment]
```
First push image to scanning repo as ***.dkr.ecr.eu-central-1.amazonaws.com/base/ubuntu:base/ubuntu-20.04-scan-1643362177
Error parsing reference: "***.dkr.ecr.eu-central-1.amazonaws.com/base/ubuntu:base/ubuntu-20.04-scan-1643362177" is not a valid repository/tag: invalid reference format
Error: Process completed with exit code 1.
```

**Comment by Andrey9kin** [comment]
fixed by #11
